ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/text_embedding/TextEmbeddingModel.java

```diff
@@ -206,7 +213,15 @@ protected void loadTextEmbeddingModel(File modelZipFile, String modelId, String
                         modelList.add(model);
 
                         Input input = new Input();
-                        input.add("warm up sentence");
+                        if (modelMaxLength != null) {
+                            StringBuilder builder = new StringBuilder();
+                            for (int j=0;j<modelMaxLength;j++) {
+                                builder.append("sentence ");
+                            }
+                            input.add(builder.toString());
```

Comment:
How about just replacing this part with one line as below?
input.add("sentence ".repeat(modelMaxLength));

Reply:
sure, good point